pkg/testutil/framework.go

```diff
@@ -232,6 +232,60 @@ func (f *Framework) NewFakeDB(name, namespace string) *fakev1alpha1.DB {
 	}
 }
 
+func (f *Framework) NewServiceCluster(name, namespace, secret string) *corev1alpha1.ServiceCluster {
```

Comment:
How about we remove the Framework pointer receiver of these helper functions and add them in https://github.com/kubermatic/kubecarrier/blob/offering-watch-api/pkg/testutil/helpers.go, the reason is because:
1. the framework receiver is unused in these helper functions, and doesn't make sense to have it.
2. Then you don't need to change the of the following offeringService, regionService, providerService in the following just for using helper functions.
3. Our unit tests can also use them in the future.

Reply:
then I will move all `NewSomething` helpers

Reply:
Yeah, would be really great, I have moved the NewProviderAccount and stuff in my account PR also.

Reply:
You can move helpers that are used by your PR, and then we move the rest of helpers gradually :)

Reply:
too late, already moved everything :)

Reply:
but I still need `func instanceService(ctx context.Context, conn *grpc.ClientConn, managementClient *testutil.RecordingClient, f *testutil.Framework) func(t *testing.T) {
` in my tests, so have to change the signatures of the following offeringService, regionService, providerService

Reply:
done